apps/native/app/src/ui/lib/field/field-card.tsx

```diff
@@ -151,7 +151,9 @@ export function FieldCard(props: FieldCardProps) {
 
   return (
     <Host>
-      {props.title !== null || (props.title !== undefined && props.code) ? (
+      {(props.title !== null &&
+        !(props.title === undefined && props.code === undefined)) ||
```

Comment:
I added this addition: `!(props.title === undefined && props.code === undefined)`. The reason for it was to skip rendering an empty header like this for firearms where both title and code are `undefined`: 
![Screenshot 2024-11-06 at 13 25 03](https://github.com/user-attachments/assets/4c88f675-2486-44a3-9997-ac311bd0eb7f).

Since the licenses are all **VERY** dynamic and I don't have access to them all on dev I did not want to change the logic that was before hand and e.g. change it to just be `!props.title ||  (props.title !== undefined && props.code)` since it theoretically could break something I don't now about ☺️